.luacheckrc

```diff
@@ -31,6 +31,7 @@ include_files = {
 exclude_files = {
     "build/**/*.lua",
     "test-run/test/test-tarantool/*.test.lua",
+    "test-run/test/unittest/*.test.lua",
```

Comment:
I think running luacheck against a git submodule is redundant. The test-run submodule itself has the own CI where all checks are run including luacheck. So my proposal is the following: replace all test-run stuff ->  `"test-run/**/*.lua"`. And please move this change to a separate commit.

Reply:
Done. However, there was no need for separate commit [before](https://github.com/tarantool/tarantool/commit/8b3ec7b928a02a8cc44aca1911cc5d665cf5133d).